rest/data/commit_queue.go

```diff
@@ -77,6 +77,19 @@ func (pc *DBCommitQueueConnector) FindCommitQueueByID(id string) (*restModel.API
 	return apiCommitQueue, nil
 }
 
+func (pc *DBCommitQueueConnector) CommitQueueRemoveItem(id, item string) (bool, error) {
+	cq, err := commitqueue.FindOneId(id)
+	if err != nil {
+		return false, errors.Wrapf(err, "can't get commit queue for id '%s'", id)
+	}
+	if cq == nil {
+		return false, nil
+	}
```

Comment:
could we just avoid returning nil, nil from FindOneId ever, to avoid this kind of awkward double error handling?

Reply:
done.